bookwyrm/templates/author/author.html

```diff
@@ -22,42 +22,59 @@ <h1 class="title">{{ author.name }}</h1>
     </div>
 </div>
 
-<div class="block content columns">
-    {% if author.aliases or author.born or author.died or author.wikipedia_link %}
+<div class="block content columns" itemscope itemtype="https://schema.org/Person">
```

Comment:
We could take the opportunity to remove the [`content`](https://bulma.io/documentation/elements/content/) which generates styles that are problematic when not used on markdown-generated code.

This would avoid adding extra margins to `<dd>` elements if we decide to use one for each alias.